### PR TITLE
Introduce test-arm PR label

### PR DESCRIPTION
**Pull request by breskeby**

### CODE REVIEW

File: .buildkite/pipelines/pull-request/part-1-arm.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+config:
+  allow-labels: "test-arm"
+steps:
+  - label: part-1-arm
+    command: .ci/scripts/run-gradle.sh -Dignore.tests.seed checkPart1
+    timeout_in_minutes: 300
+    agents:
+      provider: aws
+      imagePrefix: elasticsearch-ubuntu-2004-aarch64
```
Comment (ChrisHegarty) [inline .buildkite/pipelines/pull-request/part-1-arm.yml:9]: Is this the name of the elasticsearch image? `2004` looks a bit old!, but I suspect that the version of Ubuntu is not actually from 2004! ? ;-)

**Comment by ChrisHegarty** [comment]
This is super @breskeby. Thank you so much for adding support for testing on arm. I need it for an upcoming PR #106133.

File: .buildkite/pipelines/pull-request/part-1-arm.yml (hunk 1 above)
Comment (breskeby) [reply .buildkite/pipelines/pull-request/part-1-arm.yml:9]: no it's the ubuntu image we use. ubuntu 20.04 lts

Comment (ChrisHegarty) [reply .buildkite/pipelines/pull-request/part-1-arm.yml:9]: ha! ;-)

**Review by ChrisHegarty — APPROVED** [review]
LGTM

**Pull request merged**